src/rules/order.js

```diff
@@ -17,6 +17,16 @@ function reverse(array) {
   }).reverse()
 }
 
+function alphabetical(firstName, secondName, direction) {
+  if (direction === 'a-z') {
+    return firstName < secondName
```

Comment:
The same module could be imported twice, so this should be `<=` and `>=`. Please add a test for that case :)